data-es6.js

```diff
@@ -16662,6 +16665,7 @@ exports.tests = [
         return set instanceof S && set.has(123);
       */},
       res: {
+        closure:     true,
```

Comment:
This is full output: https://gist.github.com/teppeis/2911c795a56e8f82e95b62ce18219d55